src/gatsby-theme-material-ui-top-layout/theme.ts

```diff
@@ -225,4 +247,72 @@ const defaultTheme = createTheme({
   },
 });
 
+export const customShadows = {
+  titleShadow: `0 ${baseTheme.spacing(0.25)} ${baseTheme.spacing(
+    1
+  )} rgba(0, 0, 0, 0.5);`,
+  baseShadow: `${baseTheme.spacing(0.25)} ${baseTheme.spacing(
+    0.25
+  )} ${baseTheme.spacing(1)} rgba(0, 0, 0, 0.15)`,
+};
+
+const defaultTheme = createTheme({
+  ...baseTheme,
+  components: {
```

Comment:
Additional site-global MUI overrides can be placed here. See https://mui.com/customization/theme-components/#global-style-overrides